vllm_ascend/models/deepseek_v2.py

```diff
@@ -19,47 +19,93 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-# Adapted from
-# vllm-project/vllm/blob/main/vllm/model_executor/models/deepseek_v2.py
-# https://github.com/huggingface/transformers/blob/v4.28.0/src/transformers/models/llama/modeling_llama.py
-# vllm-project/vllm/vllm/model_executor/models/deepseek_v2.py
-"""Inference-only DeepseekV2/DeepseekV3 model."""
-from typing import Optional, Union
+# <<<<<<< HEAD
+# # Adapted from
+# # vllm-project/vllm/blob/main/vllm/model_executor/models/deepseek_v2.py
+# # https://github.com/huggingface/transformers/blob/v4.28.0/src/transformers/models/llama/modeling_llama.py
+# # vllm-project/vllm/vllm/model_executor/models/deepseek_v2.py
+# """Inference-only DeepseekV2/DeepseekV3 model."""
+# from typing import Optional, Union
+
+# import torch
+# from torch import nn
+# from transformers import PretrainedConfig
+# from vllm.config import CacheConfig, ModelConfig, VllmConfig
+# from vllm.distributed import get_pp_group, get_tensor_model_parallel_world_size
+# from vllm.model_executor.layers.fused_moe import FusedMoE
+# from vllm.model_executor.layers.layernorm import RMSNorm
+# from vllm.model_executor.layers.linear import ReplicatedLinear
+# from vllm.model_executor.layers.logits_processor import LogitsProcessor
+# from vllm.model_executor.layers.quantization import QuantizationConfig
+# from vllm.model_executor.layers.sampler import get_sampler
+# from vllm.model_executor.layers.vocab_parallel_embedding import (
+#     ParallelLMHead, VocabParallelEmbedding)
+# from vllm.model_executor.models.deepseek_v2 import (  # noqa
+#     DeepseekV2Attention, DeepseekV2DecoderLayer, DeepseekV2ForCausalLM,
+#     DeepseekV2MLAAttention, DeepseekV2MLP, DeepseekV2MoE)
+# =======
+
+import os
+from typing import Any, Dict, Optional, Union
```

Comment:
please check if this PR relate to this change https://github.com/vllm-project/vllm-ascend/pull/545